core/src/test/java/org/apache/accumulo/core/clientImpl/ClientContextTest.java

```diff
@@ -159,10 +159,9 @@ public void testGetBatchWriterConfigNotUsingDefaults() {
         .getMemoryAsBytes(ClientProperty.BATCH_WRITER_MEMORY_MAX.getValue(props));
     assertEquals(expectedMemory, batchWriterConfig.getMaxMemory());
 
-    assertEquals(Long.MAX_VALUE, batchWriterConfig.getMaxLatency(MILLISECONDS));
+    assertEquals(40, batchWriterConfig.getMaxLatency(SECONDS));
```

Comment:
Shouldn't the units here be milliseconds?

Reply:
Sorry, I may be misunderstanding the question. The TimeUnit arg to getMaxLatency could be changed to MILLISECONDS, assuming a corresponding change in _expected_ arg from 40 to 40000. But that wouldn't change the validity of the assertion, right?

Reply:
I see, the default without units is SECONDS.